Content/NPCs/Vitric/Gauntlet/FlyingGruntConstruct.cs

```diff
@@ -232,29 +234,34 @@ private void AttackBehavior()
 
 			switch (attackPhase)
 			{
-				case AttackPhase.charging:
+				case (int) AttackPhase.charging:
 
 					AnimateIdle();
 					NPC.velocity = Vector2.Lerp(NPC.velocity, direction.RotatedByRandom(0.6f) * 10, 0.05f);
+					// this is kind of rough it's a lot of consecutive frames of random movement that can't be synced normally so this will always rubberband in multiplayer unless this gets reworked to be deterministic
```

Comment:
Seems like something worth addressing, perhaps we should only have this randomization occur every say, 20 frames or so and store it in an AI field?

Reply:
going to be Trying rand seed syncing here we'll see if it works well when we hand off to testers